package.json

```diff
@@ -15,6 +15,7 @@
     "@material-ui/icons": "^4.9.1",
     "autosuggest-highlight": "^3.1.1",
     "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
```

Comment:
Can you remove bootstrap and react-bootstrap. These dependencies aren't being used anywhere in the code